plugins/standard_tests/tests/test_simple_safety_tests.py

```diff
@@ -0,0 +1,78 @@
+from newhelm.dependency_helper import FromSourceDependencyHelper
+from newhelm.placeholders import Prompt
+from newhelm.tests.simple_safety_tests import SimpleSafetyTests, SSTTestItemContext
+from newhelm.single_turn_prompt_response import (
+    MeasuredTestItem,
+    TestItem,
+    PromptWithContext,
+)
+from newhelm.typed_data import TypedData
+from tests.test_dependency_helper import MockExternalData
+
+
+def _create_mock_dependency_helper(tmpdir, prompts):
+    csv_header = ["harm_area", "category", "prompts_final_English"]
+    csv_text = "\n".join([",".join(row) for row in [csv_header, *prompts]])
+    mock_dependencies = {"prompts_csv": MockExternalData(csv_text)}
+    return FromSourceDependencyHelper(
+        tmpdir.strpath,
+        mock_dependencies,
+        required_versions={},
+    )
+
+
+def test_make_test_items(tmpdir):
+    text_prompts = [
+        [
+            '"Physical harm and Violence"',
+            "Information and advice seeking",
+            "prompt 1",
+        ],
+        ["Scams and fraud", "Instructions and actions", "prompt 2"],
+    ]
+    dependency_helper = _create_mock_dependency_helper(tmpdir, text_prompts)
+    test = SimpleSafetyTests()
+    test_items = test.make_test_items(dependency_helper)
+
+    assert len(test_items) == 2
+    assert all([len(item.prompts) == 1 for item in test_items])
```

Comment:
For at least one of the prompts, it's worth making sure the right text ends up in the right fields. Jumbling is an easy mistake to make, and a more specific test will help prevent that down the road.